tests/installation/leap2sle_zdup.pm

```diff
@@ -34,7 +34,10 @@ sub run {
 
     systemctl 'set-default --force multi-user.target';
     fully_patch_system();
-    zypper_call 'in SUSEConnect';
+    # SUSEConnect is obsoleted and provided by suseconnect-ng
+    if (script_run('rpm -q suseconnect-ng') != 0) {
+        zypper_call 'in SUSEConnect';
```

Comment:
This shouldn't be necessary anymore then

Reply:
changed and thanks a lot